R/FilterStateChoices.R

```diff
@@ -45,7 +45,7 @@ ChoicesFilterState <- R6::R6Class( # nolint
     #' \item{`"matrix"`}{ `varname` in the condition call will be returned as `<dataname>[, <varname>]`}
     #' }
     initialize = function(x,
-                          x_reactive,
+                          x_reactive = NULL,
```

Comment:
update docs to imply that the default is NULL

Reply:
Please remove this default value, we can address this later. If you change `x_reactive` in this class it would be needed also in the others. At this moment `x_reactive` must be provided and it has to be `reactive`. It could be a `reactive(NULL)`, but never NULL

Reply:
This is still not resolved, please fix before announcing that it is resolved.

Reply:
yep, sorry, commited locally but did not push